### PR TITLE
Distance sensors: allow to have sensor type for mavlink rangefinder

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

**Comment by khancyr** [comment]
Not tested yet

File: libraries/AP_RangeFinder/RangeFinder_Backend.h [hunk 1]
```diff
@@ -40,6 +40,10 @@ class AP_RangeFinder_Backend
         return sensor_type;
     }
 
+    void set_sensor_type(const MAV_DISTANCE_SENSOR type) {
```
Comment (OXINARF) [inline libraries/AP_RangeFinder/RangeFinder_Backend.h]: Please put it in _protected_.

**Comment by khancyr** [comment]
@OXINARF yep much simplier !! I will redo that !

**Comment by khancyr** [comment]
@OXINARF Ok change done ! and tested sucessfully with mavros!

**Comment by OXINARF** [comment]
Merged, thanks!

**Pull request merged**